pkg/logs/input/docker/tailer_test.go

```diff
@@ -119,7 +124,11 @@ func TestRead(t *testing.T) {
 
 func TestReadTimeout(t *testing.T) {
 	ctx, cancelFunc := context.WithCancel(context.Background())
-	tailer := NewTestTailer(&mockReaderSleep{ctx: ctx}, nil, cancelFunc)
+	reader := newMockReaderSleep(ctx)
+	// The reader should timout after testReadTimeout (10ms).
+	reader.timeout = 2 * time.Second
```

Comment:
Why is this line necessary? Won't the reader just use `testReadTimeout regardless?`